src/components/Layout/Layout.js

```diff
@@ -1,4 +1,5 @@
 import React from 'react';
+import Sidebar from 'components/Sidebar/Sidebar';
 
 const Layout = ({ noNav, children }) => {
   if (noNav) {
```

Comment:
Was wondering if you can update `noNav` to `noSideBar`. Might be clearer.

Reply:
Lets not comment on this in my PR, instead lets open an issue so we can address it in a code clean up later

Reply:
But yea good call thats a better name